## DP: Accept various mips sub-types in SUBARCH
## Upstream status: in kbuild git tree

#kbuild: Accept various mips sub-types in SUBARCH
#author	Martin Michlmayr <tbm@cyrius.com>
#	Sat, 28 Jan 2006 18:38:15 +0000 (18:38 +0000)
#committer	Sam Ravnborg <sam@mars.ravnborg.org>
#	Sat, 28 Jan 2006 21:27:39 +0000 (22:27 +0100)
#commit	780d389ab8cbd91221caa97c90cf922711076f27
#tree	f771061d0c33cbae06f7e40721762efea0140332	tree
#parent	91b06e616c62c7aef213cafb1435e94158d18901	commit | commitdiff

--- linux-2.6.15/Makefile	2006-01-16 17:56:00.000000000 +0000
+++ new/Makefile	2006-01-16 17:57:29.000000000 +0000
@@ -169,7 +169,7 @@
 SUBARCH := $(shell uname -m | sed -e s/i.86/i386/ -e s/sun4u/sparc64/ \
 				  -e s/arm.*/arm/ -e s/sa110/arm/ \
 				  -e s/s390x/s390/ -e s/parisc64/parisc/ \
-				  -e s/ppc64/powerpc/ )
+				  -e s/ppc64/powerpc/ -e s/mips.*/mips/ )
 
 # Cross compiling and selecting different set of gcc/bin-utils
 # ---------------------------------------------------------------------------
